Prepare Live Debugger docs for GA (#37664)
DataDog/documentation: content/en/tracing/live_debugger/_index.md

Live Debugger uses logpoints: auto-expiring, non-breaking breakpoints that collect diagnostic data without pausing the application. Since execution continues normally, Live Debugger can be used safely on production systems to investigate problems as they happen.

> {{< img src="tracing/live_debugger/live-debugger-demo-2025050702.mp4" alt="Live Debugger Product Demo" video="true" >}}

Reviewer: Is this mp4 file used anywhere else? If not, could you remove it from `static/images` like you did with the others?

Author: Yeah, this is still referenced by the Spanish translation (content/es/tracing/live_debugger/_index.md), so I've left it in static/images for now to avoid breaking the live /es/ page.

Reviewer: Brilliant, thanks!

## Requirements and setup

Live Debugger supports Python, Java, .NET, Ruby, Node.js, PHP, and Go. It requires the [Datadog Agent][2] (version 7.49.0 or later), an [APM-instrumented application][3], and [Remote Configuration][4].

### Enablement modes

Manage Live Debugger for each service and environment from the Live Debugger **Settings** page. Each service can be in one of three modes:

- **Automatic**: Eligible services are enabled automatically. Switch a service to **Enabled** for a faster debugging experience with debug symbols.
- **Enabled**: Live Debugger is ready to use on this service in this environment, with faster setup and a better debugging experience.
- **Disabled**: This service is explicitly disabled and is not enabled automatically.

### Minimum tracer versions

> Live Debugger requires the following minimum tracer versions:

Reviewer: One more note: for _Datadog_ tracers, we've standardized on calling them SDKs instead: https://docs.google.com/document/d/1FE7gtGc0rGSieORgro8iGi5npaElP2-ruVwFE1ZckC0/edit?tab=t.0#heading=h.yh48qfnh9cth

I'm not totally sure if this applies, because these are languages and not owned by us, but letting you know in case you want to address (can be in a followup)

The following permissions are required to use Live Debugger:

- **Live Debugger Read** (`live_debugger_read`) - Required to access the Live Debugger page.
- **Live Debugger Write** (`live_debugger_write`) - Required to create or modify Debug Sessions and logpoints.

> - **Live Debugger Redaction Write** (`live_debugger_redaction_write`) - Required to change the [redaction mode][24] for captured data.

Reviewer: ```suggestion
- **Live Debugger Read** (`live_debugger_read`): Required to access the Live Debugger page.
- **Live Debugger Write** (`live_debugger_write`): Required to create or modify Debug Sessions and logpoints.
- **Live Debugger Redaction Write** (`live_debugger_redaction_write`): Required to change the [redaction mode][24] for captured data.
```

### Protecting sensitive data

> Live Debugger data might contain sensitive information, especially when using the "Capture Variables" option. Live Debugger applies automatic mode- and identifier-based redaction to help protect sensitive data before captured data becomes available.

Reviewer: ```suggestion
Live Debugger data might contain sensitive information, especially when using the **Capture Variables** option. Live Debugger automatically applies mode-based and identifier-based redaction to help protect this data.
```

Suggested smoothing of the language here

#### Mode-based redaction

Live Debugger has two redaction modes:

- **Strict Mode**: Redacts all values except numbers and Booleans.
- **Targeted Mode**: Redacts known sensitive patterns such as credit card numbers, API keys, IPs, and other PII. It also runs a high-entropy secrets scanner that automatically redacts likely secrets, which appear as `[REDACTED:HIGH_ENTROPY]` in captured data.

> These redaction modes cannot be disabled, only switched, and Targeted Mode is applied automatically in common pre-production environments like `staging` or `preprod`. Changing the redaction mode requires the **Live Debugger Redaction Write** permission.

Reviewer: ```suggestion
These redaction modes cannot be disabled, only switched. Targeted Mode is applied automatically in common pre-production environments such as `staging` or `preprod`. Changing the redaction mode requires the **Live Debugger Redaction Write** permission.
```

Style guide preference on "like" vs. "such as," plus breaking up a long sentence.

#### Identifier-based redaction

> Variable values associated with common sensitive identifiers (for example, `password`, `accessToken`, and similar terms) are scrubbed before captured data leaves the host. Additional language-specific redaction rules are built into each tracer.

Reviewer: ```suggestion
Variable values associated with common sensitive identifiers (for example, `password`, `accessToken`, and similar terms) are scrubbed before captured data leaves the host. Each tracer includes additional language-specific redaction rules.
```

Reworded to avoid passive voice
